_includes/charts/calico/templates/calico-kube-controllers.yaml

```diff
@@ -68,7 +68,11 @@ spec:
                   key: etcd_cert
             # Choose which controllers to run.
             - name: ENABLED_CONTROLLERS
+{{- if eq .Values.network "flannel" }}
```

Comment:
I don't think this is quite right - we still want to run all these other controllers for flannel in etcd mode IIUC.

Reply:
Fixed